### PR TITLE
[86bxfj0xv][time-picker] fixed horizontal offset

### PR DESCRIPTION
**Pull request by msereniti**
## Motivation and Context

`input` tags in time picker are very narrow. It's narrow because it needs background highlight when focused. This background highlight cannot be made by `:after` or `:before` pseudo elements because `input` tag cannot contain it. 

While Select trigger is more narrow than popper, it's displayed incorrectly. To overcome it, I've added default `offset` prop that compensate it on popper.

## How has this been tested?

Our screenshot tests can't screenshot popper in right position, so only manually.

## Screenshots:

Before:

Screenshot 2024-02-22 at 16 25 17

After:

Screenshot 2024-02-22 at 16 09 23

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Nice improve.

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly or it's not required.
- [ ] Unit tests are not broken.
- [ ] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [ ] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

File: semcore/time-picker/src/PickerInput.jsx [hunk 1]
```diff
@@ -43,7 +46,7 @@ class ItemPicker extends Component {
 
   state = {
     dirtyValue: undefined,
-    visible: false,
+    visible: true,
```
Comment (ilyabrower) [inline semcore/time-picker/src/PickerInput.jsx:49]: why did you change default state value?

Comment (msereniti) [reply semcore/time-picker/src/PickerInput.jsx]: By mistake, rolled it back
